### PR TITLE
INTYGFV-14695: Added errorhandler in http403ResponseInterceptor.Service that checks if launchId caused the error.

### PR DESCRIPTION
**Pull request by ChrilleSigerhed**
Changed the test-case in MessageParserTest since I added one new message in wcMessages.

### CODE REVIEW

File: services/src/test/resources/messages/webcertMessages.js
Comment (mww-sopra-steria) [inline services/src/test/resources/messages/webcertMessages.js:22]: Is it possible to make your changes without changing the code format? Now the indentation seems to have changed.

File: web/src/main/resources/META-INF/resources/webjars/common/webcert/messages.js
Comment (mww-sopra-steria) [inline web/src/main/resources/META-INF/resources/webjars/common/webcert/messages.js:22]: Is it possible to make your changes without changing the code format? Now the indentation seems to have changed.

File: web/src/main/resources/META-INF/resources/webjars/common/webcert/utils/http403ResponseInterceptor.service.js
Comment (mww-sopra-steria) [inline web/src/main/resources/META-INF/resources/webjars/common/webcert/utils/http403ResponseInterceptor.service.js:34]: Is it possible to make your changes without changing the code format? Now the indentation seems to have changed.

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
Would be great if you can check the comments and get rid of the code-format changes. It will be easier to see what has been changed in your PR.

I will check again once it has been done.

File: web/src/main/resources/META-INF/resources/webjars/common/webcert/messages.js
Comment (ChrilleSigerhed) [reply web/src/main/resources/META-INF/resources/webjars/common/webcert/messages.js]: Indentation changed back to original format

File: services/src/test/resources/messages/webcertMessages.js
Comment (ChrilleSigerhed) [reply services/src/test/resources/messages/webcertMessages.js]: Indentation changed back to original format

File: web/src/main/resources/META-INF/resources/webjars/common/webcert/utils/http403ResponseInterceptor.service.js
Comment (mww-sopra-steria) [inline web/src/main/resources/META-INF/resources/webjars/common/webcert/utils/http403ResponseInterceptor.service.js:72]: Consider renaming the error reason 'another-session-active'. Reason for the error is because the launchId isn't valid and the webclient is therefor invalid.

Maybe just as simple as  'invalid-launchid'.

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
I'm thinking if you should make adjustment on the message that the server provides in the filter. Good if you wait with merging this until the Webcert PR is also ready for merge.

File: web/src/main/resources/META-INF/resources/webjars/common/webcert/utils/http403ResponseInterceptor.service.js
Comment (mww-sopra-steria) [inline web/src/main/resources/META-INF/resources/webjars/common/webcert/utils/http403ResponseInterceptor.service.js:71]: Probably will require a change based on the outcome from the webcert PR.

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
See comment. If no change is required, just as for a new review.

**Review by mww-sopra-steria — APPROVED** [review]
Approved! Just wait for @mikeyOne to merge his PR first in common before you merge this.

**Pull request merged**
